solr_helpers/__init__.py

```diff
@@ -556,7 +558,9 @@ def build_solr_query(filters, comb_with='OR', with_tags_for_ex=False, subq_join_
             count += 1
 
         query_set[attr_name] = query_str
-
+    #filter_tags ==None in IDC
+    i=1
+    pass
```

Comment:
I'm unclear what this is meant to accomplish? We definitely tag filters in IDC--that's how we do exclusions for counting purposes.

Reply:
I just had this in as breakpoints